otx/algorithms/common/tasks/training_base.py

```diff
@@ -126,6 +128,20 @@ def __init__(self, task_config, task_environment: TaskEnvironment, output_path:
         # to override configuration at runtime
         self.override_configs = {}  # type: Dict[str, str]
 
+        if self._is_multi_gpu_training():
+            self._setup_multigpu_training()
+
+    @staticmethod
+    def _is_multi_gpu_training():
+        return "LOCAL_RANK" in os.environ
+
+    @staticmethod
+    def _setup_multigpu_training():
+        if not dist.is_initialized():
```

Comment:
Not sure, how about
```suggestion
        if not dist.is_initialized() and torch.cuda.is_available():
```

Reply:
Nice catch! but I think checking it in `_is_multi_gpu_training()` is better. I'll update it.

Reply:
I fixed.